posthog/tasks/status_report.py

```diff
@@ -17,22 +18,29 @@ def status_report(*, dry_run: bool = False) -> Dict[str, Any]:
     period_start, period_end = get_previous_week()
     report: Dict[str, Any] = {
         "posthog_version": VERSION,
+        "deployment": os.environ.get("DEPLOYMENT", "unknown"),
         "period": {"start_inclusive": period_start.isoformat(), "end_inclusive": period_end.isoformat()},
     }
+
     report["users_who_logged_in"] = [
         {"id": user.id, "distinct_id": user.distinct_id}
         if user.anonymize_data
         else {"id": user.id, "distinct_id": user.distinct_id, "first_name": user.first_name, "email": user.email}
```

Comment:
I know this is unrelated, but adding it here as this PR is an improvement on this code. I don't think it ever makes sense to send the user's first name and email, we're just adding PII here that it'll be very hard to keep track of. With the distinct ID we should be able to get their name if they haven't anonymized their data.

Also maybe send the total number of registered users too?

Reply:
Please file a separate issue with a suggestion how to anonymize. :)

Reply:
Sounds good, I'll make a PR for this after this one is merged